### PR TITLE
whew

### PR DESCRIPTION
**Pull request by jvinal**

### CODE REVIEW

File: client/html/pages/tom_clancy.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <div id="tom_clancy">
+	<audio autoplay="autoplay" loop="loop"><source src="wup.mp3" type="audio/mpeg"/></audio>
```
Comment (legauntt) [inline client/html/pages/tom_clancy.html:2]: File doesn't exist in build - Was this tested?

**Comment by jvinal** [comment]
@Pancakeo @TheGauntt

File: client/html/pages/tom_clancy.html (hunk 1 above)
Comment (legauntt) [reply client/html/pages/tom_clancy.html:2]: Also, please run "npm run prod" (from client/) and commit resulting dist/

**Review by legauntt — CHANGES_REQUESTED** [review]

File: client/html/pages/tom_clancy.html (hunk 1 above)
Comment (jvinal) [inline client/html/pages/tom_clancy.html]: The file is uploaded as part of the pull request.

File: client/dist/html/pages/tom_clancy.html [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <div id="tom_clancy">
+	<audio autoplay="autoplay" loop="loop"><source src="wup.mp3" type="audio/mpeg"/></audio>
```
Comment (legauntt) [inline client/dist/html/pages/tom_clancy.html:2]: Don't modify dist/, let the build script do that.

**Review by legauntt — CHANGES_REQUESTED** [review]

File: client/dist/html/pages/tom_clancy.html (hunk 2 above)
Comment (jvinal) [reply client/dist/html/pages/tom_clancy.html:2]: That wasn't me; the only thing the build script did to dist/ was apply the exact changes that were in client/html/pages.

Comment (legauntt) [reply client/dist/html/pages/tom_clancy.html:2]: I see - there's no JS/CSS file so it keeps the hashes the same. This is a great improvement - the only problem is, it doesn't work.

👎

**Comment by legauntt** [comment]
You'll want to use either the public/ folder, or resolve the mp3 correctly. Please test all changes prior to submitting a PR. 💯

**Pull request merged**